kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/protocolfilter/KafkaServiceReconcilerTest.java

```diff
@@ -70,7 +70,7 @@ void shouldMarkFilterAccepted() {
                                                 .isEqualTo(OBSERVED_GENERATION);
                                         assertThat(kafkaServiceStatus)
                                                 .singleCondition()
-                                                .hasObservedGenerationInSyncWithMetadataOf(input.getResource().get())
+                                                .hasObservedGenerationInSyncWithMetadataOf(kafkaService)
```

Comment:
Oops this makes me notice we have KafkaServiceTest (at least but probably KafkaService) in the wrong package.

Reply:
Yeah, I noticed that you put the KSReconciler in its own package. IDK whether that's the right thing to do or not. Probably it is, but I think we need to see how the Reconcilers interact with each other. 

Shall I move it into the same package as the other for now, just for consistency? And move them all later on when we know how they interrelate?

Reply:
I think we were expecting to put a few other things in the package so happy to move it now.